Packs/ContentManagement/Scripts/CommitFiles/CommitFiles.yml

```diff
@@ -1,4 +1,8 @@
 args:
+- description: Should the script keep all placeholders (${}) in the files or replace them with the data from the context.
```

Comment:
```suggestion
- description: Set to true if the files should be committed while preserving ${} placeholders instead of replacing them with context data.```